src/views/Contribute/Contribute.tsx

```diff
@@ -0,0 +1,311 @@
+import {
+  Accordion,
+  Stack,
+  OrderedList,
+  ListItem,
+  Text,
+  UnorderedList,
+} from "@chakra-ui/react";
+import type { FunctionComponent } from "react";
+import { FAQ, FAQHeading, FAQItem, FAQLink } from "../../components/FAQ";
+import { Page } from "../../components/Page";
+
+export const Contribute: FunctionComponent = () => (
+  <Page
+    meta={{
+      description: "Learn about contributing to Construct Hub.",
+      title: "Contribute",
+    }}
+    pageName="contribute"
+  >
+    <FAQ>
+      <FAQHeading>Contributing to Construct Hub</FAQHeading>
+
+      <Accordion allowMultiple allowToggle defaultIndex={[0]}>
```

Comment:
Would it make sense to use `FAQSections` here? Also I think `allowToggle` is enabled by default (could be wrong)

Reply:
Not really, since `FAQSections` is the parent component we use on `views/FAQ` to give a higher-level heading such as `General`